### PR TITLE
Fixed megastore entries in alt map key mod

### PR DESCRIPTION
**Pull request by ZhilkinSerg**
#### Summary

```SUMMARY: None```

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
Mall? Do you mean megastore?

**Comment by ZhilkinSerg** [comment]
Yes, thanks. I am on mobile, so can't reword commit message though.

File: data/mods/alt_map_key/overmap_terrain.json [hunk 1]
```diff
@@ -840,20 +840,159 @@
...
+    "id": "megastore_0_2_0",
+    "copy-from": "megastore_0_0_0"
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_1_2_0",
+    "copy-from": "megastore_1_2_0",
     "name": "megastore",
-    "sym": "m",
+    "sym": "+",
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
    "sym": "m",
```

File: data/mods/alt_map_key/overmap_terrain.json [hunk 2]
```diff
@@ -840,20 +840,159 @@
...
+    "type": "overmap_terrain",
+    "id": "megastore_2_2_1",
+    "copy-from": "megastore_0_0_1"
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_0_0_roof",
+    "copy-from": "megastore_0_0_roof",
+    "name": "megastore roof",
+    "sym": "M",
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
    "sym": "m",
```

File: data/mods/alt_map_key/overmap_terrain.json [hunk 3]
```diff
@@ -840,20 +840,159 @@
...
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_2_2_1",
+    "copy-from": "megastore_0_0_1"
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_0_0_roof",
+    "copy-from": "megastore_0_0_roof",
+    "name": "megastore roof",
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
```

File: data/mods/alt_map_key/overmap_terrain.json [hunk 4]
```diff
@@ -840,20 +840,159 @@
...
+    "copy-from": "megastore_parking",
+    "name": "megastore parking lot",
+    "sym": "O",
+    "color": "dark_gray"
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_0_0_0",
+    "copy-from": "megastore_0_0_0",
     "name": "megastore",
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
```

File: data/mods/alt_map_key/overmap_terrain.json [hunk 5]
```diff
@@ -840,20 +840,159 @@
   },
   {
     "type": "overmap_terrain",
-    "id": "megastore_entrance",
-    "copy-from": "megastore_entrance",
+    "id": "megastore_parking",
+    "copy-from": "megastore_parking",
+    "name": "megastore parking lot",
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
```

File: data/mods/alt_map_key/overmap_terrain.json [hunk 6]
```diff
@@ -840,20 +840,159 @@
...
+    "sym": "O",
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
    "sym": "p",
```

File: data/mods/alt_map_key/overmap_terrain.json [hunk 7]
```diff
@@ -840,20 +840,159 @@
...
+    "color": "dark_gray"
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
    "color": "light_blue"
```

File: data/mods/alt_map_key/overmap_terrain.json [hunk 8]
```diff
@@ -840,20 +840,156 @@
...
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_0_2_0",
+    "copy-from": "megastore_0_0_0"
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_1_2_0",
+    "copy-from": "megastore_1_2_0",
     "name": "megastore",
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
```

File: data/mods/alt_map_key/overmap_terrain.json [hunk 9]
```diff
@@ -840,20 +840,156 @@
...
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_2_2_0",
+    "copy-from": "megastore_0_0_0"
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_0_0_1",
+    "copy-from": "megastore_0_0_1",
+    "name": "megastore, second floor",
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
```

File: data/mods/alt_map_key/overmap_terrain.json [hunk 10]
```diff
@@ -840,20 +840,156 @@
...
+    "sym": "M",
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
    "sym": "m",
```

File: data/mods/alt_map_key/overmap_terrain.json [hunk 11]
```diff
@@ -840,20 +840,154 @@
...
+    "id": "megastore_parking",
+    "copy-from": "megastore_parking",
+    "sym": "p",
+    "color": "light_blue"
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_0_0_0",
+    "copy-from": "megastore_0_0_0",
     "sym": "M",
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
    "sym": "m",
```

File: data/mods/alt_map_key/overmap_terrain.json [hunk 12]
```diff
@@ -840,20 +840,154 @@
...
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_2_2_1",
+    "copy-from": "megastore_0_0_1"
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_0_0_roof",
+    "copy-from": "megastore_0_0_roof",
     "sym": "m",
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
```

File: data/mods/alt_map_key/overmap_terrain.json [hunk 13]
```diff
@@ -840,20 +840,154 @@
...
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_2_2_0",
+    "copy-from": "megastore_0_0_0"
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_0_0_1",
+    "copy-from": "megastore_0_0_1",
+    "sym": "m",
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
```

File: data/mods/alt_map_key/overmap_terrain.json [hunk 14]
```diff
@@ -840,20 +840,154 @@
...
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_0_2_0",
+    "copy-from": "megastore_0_0_0"
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "megastore_1_2_0",
+    "copy-from": "megastore_1_2_0",
+    "sym": "m",
```
Comment (ZhilkinSerg) [inline data/mods/alt_map_key/overmap_terrain.json]:
```suggestion
```

**Comment by Karthas077** [comment]
I'm pretty sure this still needs to be merged.

**Pull request merged**